requirements/deploy.txt

```diff
@@ -76,7 +76,7 @@ click-repl==0.2.0
     # via
     #   -r requirements/pip.txt
     #   celery
-cryptography==40.0.2
+cryptography==41.0.1
```

Comment:
> BACKWARDS INCOMPATIBLE: Support for OpenSSL less than 1.1.1d has been removed. Users on older version of OpenSSL will need to upgrade.

This is for our own application, where we are using Ubuntu 22.04. So, we should be fine here.